.github/workflows/es-assets.yml

```diff
@@ -0,0 +1,13 @@
+name: Check Endless Sky Low DPI Assets
+on:
+  push:
+  pull_request:
```

Comment:
If we can manually trigger this action, do we need to run it automatically for PRs? Since it will always error out until the main PR is merged.

Reply:
I defer to this repo to adjust whatever is necessary.  It can be removed if you want